### DISCUSSION TITLE
layout部分配置不生效

### DISCUSSION THREAD

**Discussion by singgl** [umijs/umi#11527] [General]
![image](https://github.com/umijs/umi/assets/26128398/6988c830-fed2-4a6b-8e72-777902db9edb)
![image](https://github.com/umijs/umi/assets/26128398/24c77d7b-1f3f-499e-9f2b-656155f2da2b)
配置在config.ts中不生效,直接配置在layout上才生效

**Comment by fz6m** [comment]
如有问题，需要给一个最小复现。
